File: 03_Manipulating_Items/01_PutItem.cs
AmazonDynamoDBClient client = new AmazonDynamoDBClient();
string tableName = "productTable";

var request = new PutItemRequest
{
   TableName = tableName,
   Item = new Dictionary<string, AttributeValue>()
      {
          { "id", new AttributeValue { N = "20" }},
          { "type", new AttributeValue { S = "book" }},
          { "stock", new AttributeValue { N = "110" }},
          { "price", new AttributeValue { N = "55" }},
          { "title", new AttributeValue { S = "Mastering DynamoDB" }},
          
      }
};

client.PutItem(request);

File: 03_Manipulating_Items/01_putItem.js

//with docementClient
var params = {
    TableName: 'productTable',
    Item: { // a map of attribute name to AttributeValue
        
        id: 51,
        type:"book",
        stock: 110,
        price:20,
        title: "test book"
    },
    ReturnConsumedCapacity: "TOTAL",
    ReturnValues: "ALL_OLD"
};
docClient.put(params, function(err, data) {
    if (err) ppJson(err); // an error occurred
    else ppJson(data); // successful response
});